.github/workflows/status-embed.yml

```
@@ -0,0 +1,56 @@
+name: Status Embed
+
+on:
+  workflow_run:
+    workflows:
+      - Test & Lint
+    types:
+      - completed
+
+jobs:
+  status_embed:
+    if: github.event.workflow_run.conclusion != 'skipped'
+    name:  Send a Status Embed to Discord
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get Pull Request Information
+        id: pr_info
+        if: github.event.workflow_run.event == 'pull_request'
+        run: |
+          curl -s -H "Authorization: token $GITHUB_TOKEN" ${{ github.event.workflow_run.artifacts_url }} > artifacts.json
```

Likely a good idea to add these. `-S` shows errors even when silent. `-L` follows HTTP 3xx redirects.
```suggestion
          curl -sSL -H "Authorization: token $GITHUB_TOKEN" ${{ github.event.workflow_run.artifacts_url }} > artifacts.json
```